scipy/stats/tests/test_hypotests.py

```diff
@@ -1482,6 +1425,20 @@ def test_engineering_stat_handbook(self):
             assert_allclose(lower[i, j], conf.low[i, j], atol=1e-2)
             assert_allclose(upper[i, j], conf.high[i, j], atol=1e-2)
 
+    def test_rand_symm(self):
+        np.random.seed(1234)
+        data = np.random.rand(3, 100)
+        res = stats.tukey_hsd(*data)
+        conf = res.confidence_interval()
+        # the confidence intervals should be negated symmetric of each other
+        assert_allclose(conf.low, -conf.high.T)
+        # upper diagonals of statistic should be negative symmetric to lower
+        # diagonals
+        assert_allclose(np.tril(res.statistic), -np.triu(res.statistic).T)
```

Comment:
```suggestion
        assert_allclose(res.statistic, -res.statistic.T)
```
Same, right?

Reply:
> First line is similar to test of confidence intervals, right?
Should check the diagonal, too.

I assume you mean to check that the center diagonals are constant, since a self comparison should have 0 mean difference?

Reply:
Right.